superset-frontend/src/views/CRUD/welcome/ActivityTable.tsx

```diff
@@ -79,31 +97,68 @@ const ActivityContainer = styled.div`
   }
 `;
 
+const UNTITLED = t('[Untitled]');
+const UNKNOWN_TIME = t('Unknown');
+// translation keys for last action on
+const TRANS_LAST_VIEWED = `Last viewed %s`;
+const TRANS_LAST_MODIFIED = `Last modified %s`;
+
+const getEntityTitle = (e: ActivityObject) => {
```

Comment:
I know this is just a refactor, but `e` is a pretty bad variable name (i read it as `error` or `event` first, not `entity`). Can we rename throughout to `entity`?

Reply:
Sure!